packages/core/src/behavior.ts

```diff
@@ -54,8 +57,8 @@ export interface Behavior<TReceived extends EventObject, TEmitted> {
   subscribe?: (observer: Observer<TEmitted>) => Subscription | undefined;
 }
 
-export function createCallbackBehavior<TEvent extends EventObject>(
-  callback: InvokeCallback,
+export function createDeferredBehavior<TEvent extends EventObject>(
```

Comment:
In the future, this could use heuristics for lazily invoked promises/observables/machines/actors. (to be discussed)